### ISSUE TITLE
Main Page Examples are Broken

### ISSUE THREAD

**Issue by meawoppl** [FNNDSC/ami#295]
### Description
This link
http://jsfiddle.net/gh/get/library/pure/fnndsc/ami/tree/master/lessons/00#run
Is on the main page and ends up with a error 422?  

#### Live Demo
Its in master above.

#### Steps to Reproduce
Follow the link

#### Expected Results

Loader example in jsfiddle as in the past.

### Browsers Affected
- [X] All
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari 9
- [ ] Safari 8
- [ ] IE 11

### Versions
- AMI: vX.X.X
- THREEJS: vX.X.X

**Comment by NicolasRannou** [comment]
Thanks for reporting, I suspect that the ami.js build file provides some features that jsfiddle doesn't support. One thing that may fix it is change the babel targets in https://github.com/FNNDSC/ami/blob/dev/.babelrc

to be tested

**Comment by PtiLuky** [comment]
With the 422 error, I have a  "Js wrap is invalid" message.
it seems that in [demo.details](https://github.com/FNNDSC/ami/blob/dev/lessons/00/demo.details) _js_wrap: onLoad_ is the problem ?
A possible solution : (?)
```
js_wrap: l
```
https://github.com/jsfiddle/jsfiddle-issues/issues/949

I don't know how to try it (without control on it) (and why would it stop working without any change... ?)

**Comment by NicolasRannou** [comment]
@PtiLuky the fix you suggested works, can you please submit a PR?

**Comment by kuige** [comment]
lessons 00 is ok now, but the other lessons start from 01 are still broken, 422 error, "Js wrap is invalid"

**Comment by zeigerpuppy** [comment]
I can confirm, lesson 00 is working but the other still result in `Js wrap is invalid` error.
OS X 10.11.6, Firefox v61.0.1

**Comment by NicolasRannou** [comment]
Will be fixed soon, once #dev is merged into master.

Then we will just work on the master branch for a faster development process.
